### ISSUE TITLE
Critical - Cannnot create an account after successful preapp submit?

### ISSUE THREAD

**Issue by transystems** [TransystemsLLC/ContactApp-4.6.1#9]
http://transystemsllc.com/#/apply/register

Fill in the email and both password fields. Create account button is enabled as expected. When you click the create account button, the page is cleared of data but no login and redirect occurs as expected.

No database entry is posted.

![image](https://cloud.githubusercontent.com/assets/16327097/25417091/c79116ac-29fe-11e7-88e6-29ab99a569d3.png)

**Comment by TomasPDL** [comment]
So my code was updated on the FTP, but not the actual GitHub, so your changes overwrote mine- that's my fault.

I went ahead and pulled your changes and re-added the fixes I had done, but since the connection string files are filtered under .gitignore, my publishes overwrote any connection string changes you might have made, bringing up the first issue again.

I'm pushing my changes up to the github proper now, so if you overwrite connection files both pieces should work as intended.
Many apologies!
